model/polos.php

```diff
@@ -25,7 +25,7 @@
       $rows[]=$row;
   }
 }
-elseif ($operation == "polosbystate") {
+elseif ($operation == "polosbystatewithfederativeunit") {
```

Comment:
Se não tiver algo intencional, recomendo usar `===` https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons